### PR TITLE
Add SDL to strawberry command from awesome work of @srbiotik

### PR DESCRIPTION
**Pull request by Ambro17**
## Description

Given an SDL file, when one runs 
`strawberry import-schema <SDL_FILE_PATH> > schema.py`
One gets autogenerated code equivalent to the sdl dumped into `schema.py` file

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [x] New feature
- [ ] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

#159 

## Checklist

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by botberry** [comment]
Thanks for adding the `RELEASE.md` file!



Here's a preview of the changelog:

---

Adds import-schema command to cli import-schema
The command transpiles SDL from a path to strawberry types which are printed to the stdout to be used at will.
Call it by: `strawberry import-schema <PATH>`


---
Here's the preview release card for twitter:




Here's the tweet text: 
```
🆕 Release (next) is out! Thanks to Nahuel Ambrosini for the PR 👏

Get it here 👉 https://github.com/strawberry-graphql/strawberry/releases/tag/(next)
```

File: strawberry/cli/commands/schema_importer/sdl_transpiler.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    """
+    Format and return a whole attribute string
+    consists of attribute name in snake case and field type
+    """
+    field_name = get_field_name(field.name.value)
+    field_type = get_field_type(field)
+    is_custom_type = not any(
+        builtin_type in field_type
+        for builtin_type in ("str", "int", "float", "ID", "bool")
+    )
```
Comment (BryceBeagle) [inline strawberry/cli/commands/schema_importer/sdl_transpiler.py:121]: Am I correct in understanding that `field_type` is a string? Any custom type with any of these as a _substring_ (e.g. if I had a type named "winter" or something), would make `is_custom_type == False` no?

Comment (Ambro17) [reply strawberry/cli/commands/schema_importer/sdl_transpiler.py:121]: You're correct. Given a string like `List[int]` it tries to detect if it uses a custom type with this logic. But it does have a problem you just found. Can you think of a better solution?

Comment (BryceBeagle) [reply strawberry/cli/commands/schema_importer/sdl_transpiler.py:121]: Sorry for the delayed response here. Could you do something like `field_type == "int" or "[int]" in field_type` (obviously expanded for all the types here)?

**Pull request closed without merging**